src/shaderlib/picking/picking.js

```diff
@@ -1,17 +1,15 @@
-const DEFAULT_HIGHLIGHT_COLOR = new Uint8Array([0, 64, 128, 64]);
-
 const DEFAULT_MODULE_OPTIONS = {
   pickingSelectedColor: null, //  Set to a picking color to visually highlight that item
-  pickingHighlightColor: DEFAULT_HIGHLIGHT_COLOR, // Color of visual highlight of "selected" item
```

Comment:
removed `pickingHighlightColor`, ideally this can only be set once and doesn't have to be updated as frequently as other uniforms. So when other uniforms are set, we don't overwrite this with default value.